Cli.py

```diff
@@ -85,6 +89,8 @@ def session(self):
             session_id_record=self.cursor_1.fetchall()
             for i in session_id_record:
                 self.session_id=i[0]
+                #list comp maybe
```

Comment:
Yes, I think so! What might that look like?

Reply:
Actually I'm not sure why this is here. Why is the same variable assigned multiple times? It only keeps the last value assigned to it. Is that desired?